pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md

```diff
@@ -0,0 +1,108 @@
+
+Level one & level two contributions share components between vendors. If there is a component fulfills a purpose similar to your business, consider reusing that component. This requires much less effort and is more long-term maintainable.
+
+After deciding what level of contribution meets your use case, open a [GitHub issue](https://github.com/ampproject/amphtml/issues/new) to start. 
+
+
+## Level one contribution
+
+Level one contributions share components and load vendor specific logic as custom JavaScript in a cross origin iframe. 
+
+Vendors add configurations or features to existing extension to implement their functionalities. If such a component does not exist they may propose a new one. 
+
+The only vendor specific logic checked into AMP repo is a vendor configuration. Adding a new vendor to an existing level one contribution typically does not need a design review. Vendors can follow the documentation of the component.
+
+### Vendor expectations
+
+*   Maintain and serve vendor’s custom JavaScript independently.
+*   Provide tests for their configuration and respond to issues. 
+*   Provide a troubleshooting channel for developers. 
+*   Respond to any and all bug filings related to their service. 
+
+### Level one example
+
+[**amp-ad**](../../../components/reference/amp-ad.md)
+
+If you are an ad technology provider looking to integrate with AMP, read the [development overview](https://github.com/ampproject/amphtml/tree/master/ads#overview) or jump to the [developer instructions](https://github.com/ampproject/amphtml/tree/master/ads#developer-guidelines-for-a-pull-request) for adding your support to [`amp-ad`](../../../components/reference/amp-ad.md). Depending on the ad technology your company provides, you might find [these integration instructions ](/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/ad-integration-guide.md?format=ads)useful.
+
+There are 90+ ad providers who have added support for advertising related features like amp-ad. Here is a [sample pull request](https://github.com/ampproject/amphtml/pull/2299) from the ad network [Criteo](https://github.com/ampproject/amphtml/blob/master/ads/criteo.md).
+
+## Level two contribution 
+
+Level two vendor contributions share components. All logic is checked into the AMP repository, and no custom Javascript can be loaded into an iframe.  Vendors add configurations or features to existing components to implement their functionalities. If such a component does not exist they may propose a new one. 
+
+All business logic is checked into AMP repo, but the only vendor specific logic checked in is a vendor configuration. If component works with a vendor provided config file, no design review is needed. If the vendor configuration implements a new feature or new component it will need to pass AMP’s design review. 
```

Comment:
Add an example for business logic and vendor specific logic to make it more clear what these mean.